app/src/main/java/com/keylesspalace/tusky/entity/TrendingTagsResult.kt

```diff
@@ -0,0 +1,29 @@
+/* Copyright 2023 Tusky Contributors
+ *
+ * This file is a part of Tusky.
+ *
+ * This program is free software; you can redistribute it and/or modify it under the terms of the
+ * GNU General Public License as published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Tusky is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even
+ * the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
+ * Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with Tusky; if not,
+ * see <http://www.gnu.org/licenses>. */
+
+package com.keylesspalace.tusky.entity
+
+data class TrendingTag(
+    val name: String,
+    val url: String,
```

Comment:
I know it's inconsistent with the rest of the code, but would you mind adding one/two line Javadoc here explaining the semantics of the different fields. I know it's in the API docs, but it's tedious to have to go and look that up when it could appear inline in Android Studio.

For example:

```kotlin
[...]
/** URL to a page (not an API call) on the server to view statuses with this tag */
var url: String
[...]
```

I want to slowly clean this up in the existing API implementation, so may as well start here.

Reply:
No problem at all. That makes complete sense.

:+1: Resolved locally.